src/contexts/account-context.tsx

```diff
@@ -45,7 +45,7 @@ export const AccountContextProvider: React.FC<PropsWithChildren> = ({ children }
     const { data, error } = await accountService.getAccount(getAccessToken());
     if (data) {
       setAccountDetails(new Account({ ...data }));
-      if (!data.firstName) {
+      if (!data.first_name) {
```

Comment:
Why are we changing this? In JS, we want to use camelCase for variable name.

Reply:
The idea is:

- Service layer makes a request to server and gets a JSON from servicer where each type is primitive. Ex: date is returned as string.
- It converts the object into JS object (not JSON) by converting JSON into strong type. ex: converts string to date when a date is returned
- Context interacts with servicer to get the JS object (which should follow JS naming conventions)
- UI always talks to context which maintains the state of the data

Reply:
`const { data, error } = await accountService.getAccount(getAccessToken());` should return `account` of type `Account`, which should `firstName`

The problem is service layer needs to convert JSON from server into JS object.